contrib/plugin-gallery/static/gallery-footer.html

```diff
@@ -4,7 +4,7 @@
 	<footer>
 		<p class="tagline">This page was generated by <a href="http://munin-monitoring.org/wiki/PluginGallery">gallery-build.sh</a>.</p>
 	</footer>
-	<script src="/static/js/jquery-1.12.4.js"></script>
+	<script src="/static/js/jquery-3.7.1.min.js"></script>
```

Comment:
Can we have the non-minified version ?

Reply:
On https://jquery.com/download/ they call them "production" versus "development", so I went with "production". 🤷‍♀️